node/src/reactor/participating.rs

```diff
@@ -733,20 +733,24 @@ impl reactor::Reactor for Reactor {
                 let block_header = match storage.read_block_by_height(height) {
                     Ok(Some(block)) => block.header().clone(),
                     Ok(None) => {
-                        error!(
+                        warn!(
                             "unable to retrieve block at height {} for migration to rocksdb",
                             height,
                         );
                         continue;
                     }
                     Err(err) => {
-                        error!(
+                        warn!(
                             "unable to retrieve parent block at height {} for migration to rocksdb {:?}",
                             height, err,
                         );
                         continue;
                     }
                 };
+                info!(
+                    "Will migrate state root {:#?} at height {}",
+                    state_roots, height
+                );
```

Comment:
if this is using the tracing crate then it should be:
```rust
info!(?state_roots, ?height, "will migrate state root at height")
```

Reply:
Fixed in 213bf27a50f00424def44d1557567f3f3d184e21